assets/cubyz/biomes/spawn.zig.zon

```diff
@@ -18,7 +18,6 @@
 			.id = "cubyz:simple_tree",
 			.leaves = "cubyz:oak_leaves",
 			.log = "cubyz:air",
-			.top = "cubyz:oak_top",
```

Comment:
This also changes the behavior, now the spawn trees will be pure leaf balls.
It's not that relevant though, since it is quite a rare biome and I don't care about this one detail.